server/googleActions.js

```diff
@@ -129,8 +142,8 @@ const GoogleActions = {
 	  return brews;
 	},
 
-	existsGoogleBrew : async (auth, id)=>{
-		const drive = google.drive({ version: 'v3', auth: auth });
+	existsGoogleBrew : async (id)=>{
```

Comment:
I think this whole function can be deleted. We only use it to check if a brew exists before submitting an update, but `updateGoogleBrew` should already be handling that with an error response if we try to update a brew that doesn't exist.